where-to-travel/src/main/webapp/script.js

```diff
@@ -259,13 +259,12 @@ function addPlacesFromDirection(lat, lng, place_candidates) {
 
         function callback(response, status) {
           if (status == 'OK') {
-            const origins = response.originAddresses;
             for (row of responce.rows) {
-              const results = response.rows[row].elements;
+              const results = row.elements;
               for (result of results) {
-                const element = results[result];
+                const element = result;
                 //Check if the time is within the +- 20% of the user's requested time
```

Comment:
We should check that the status of the element is ok first ("if (element.status == 'OK')") in case any of the destinations we provide to the api are inaccessible (i.e. on a nearby island)